### DISCUSSION TITLE
Customization contributions-- PR or fork?

### DISCUSSION THREAD

**Discussion by miduncan** [Comcast/react-data-grid#2770] [General]
Hey all,

Had a quick question to the maintainers-- I see that the commit history is mostly between a few people. My team at Microsoft is currently looking at something like this, but we would need to make some options for additional customizations (for example, an option to change what keys navigate the grid).  Would contributions such as this be welcome, or would the advice be to fork?

**Comment by miduncan** [comment]
@amanmahajan7 @nstepien

**Comment by nstepien** [comment]
We're open to contributions, we just haven't had the time to devote ourselves more to maintain the lib this past year, hopefully 2022 will be different.
